packages/w3up-client/test/client.test.js

```diff
@@ -259,7 +259,33 @@ export const testClient = {
       assert.equal(spaces.length, 1)
       assert.equal(spaces[0].did(), space.did())
     },
-  },
+
+    'should create a space with recovery account': async (assert, { client, mail, grantAccess }) => {
+      const account = client.login('alice@web.mail')
+
+      await grantAccess(await mail.take())
+
+      const alice = await account
+
+      const space = await client.createSpace('recovery-space', {
+        account: alice,
+      })
+      assert.ok(space)
+
+      const proof = alice.agent.proofs()
+      .find(p => p.issuer.did() === alice.did())
+      if (!proof) {
+        throw new Error('Recovery Proof not found')
+      }
+      
+      assert.ok(proof)
+      assert.equal(proof.audience.did(), alice.did())
```

Comment:
Thanks for the feedback @hannahhoward. I've applied the suggested changes. What is the best way to check if that account is a recovery account?